esp/services/ws_machine/ws_machineService.cpp

```diff
@@ -2848,29 +2846,30 @@ bool Cws_machineEx::onGetComponentUsage(IEspContext& context, IEspGetComponentUs
 
         StringBuffer timeStr;
         IArrayOf<IEspComponentUsage> componentUsages;
-        Owned<IPropertyTree> usageReq = createPTree("Req");
+        Owned<IEnvironmentFactory> envFactory = getEnvironmentFactory(true);
+        Owned<IConstEnvironment> constEnv = envFactory->openEnvironment();
+        Owned<IPropertyTree> usageReq = readComponentUsageReq(req, constEnv);
```

Comment:
minor: now this is a getter method and there are other readXX methods that don't return, it would be clearer if this and other similar methods were renamed getComponentUsageReq